packages/gatsby-source-graphql/README.md

```diff
@@ -0,0 +1,154 @@
+# gatsby-source-graphql
+
+Plugin for connecting arbitrary GraphQL APIs to Gatsby GraphQL. Remote schemas are stitched together by adding a type that wraps the remote schema Query type and putting it under field of Gatsby GraphQL Query.
+
+- [Example website]([See it here](https://using-gatsby-source-graphql.netlify.com/)
```

Comment:
This isn't live yet?

Reply:
Yeah, turns out I need to be admin of this repo to add it to netlify.

Reply:
@freiksenet you can add a netlify site without granting access to the repo: https://github.com/netlify/netlifyctl/blob/master/README.md#manual-setup